### ISSUE TITLE
Removing  Microsoft.Phone.Controls.Toolkit and Coding4Fun.Toolkit Dependency

### ISSUE THREAD

**Issue by writeosahon** [leecrossley/cordova-plugin-datetime-picker#6]
Hello @leecrossley . 
Your plugin is exactly what I need for my Windows Phone PhoneGap/Cordova app. The screenshot is a thing of beauty!! :) 
Unfortunately, i noticed that the plugin requires 2 dependencies. This means i cannot build my app with your plugin on Phonegap Build :(
How soon do you expect to solve the dependency problems and host the plugin on a public registry?

Thanks for the help

**Comment by leecrossley** [comment]
The best solution may be to include these binaries in the plugin

**Comment by writeosahon** [comment]
Hi @leecrossley thanks for the response.
If that's the best solution, then go for it !! :)
They are a couple of datepicker plugins for Android and iOS, but none for Windows Phone.
It would be nice to have this in cordova and phonegap plugin registries.
How soon can you work on this??
Hope to hear from you soon.

PS: nice work on building the plugreg directory.
Cheers

**Comment by writeosahon** [comment]
Hi @leecrossley still no word on including the necessary binary dependencies and posting this plugin to cordova registry.
Please, are you still working on this??

Cheers

**Comment by leecrossley** [comment]
Yes, but I am very busy I'm afraid and paying clients take priority. It is on my list and will update here when I've had a chance to implement.

**Comment by mletynski** [comment]
Is it possible the build the project with cordova build ? I installed missing dependencies via Visual Studio but i don't know how to "tell" cordova build to use those two libs. Any hint ?

**Comment by sumitsoman** [comment]
Has there been an update on this issue? I'm also unable to build the project after adding this plugin.

![image](https://cloud.githubusercontent.com/assets/5772710/20129979/ab27c960-a679-11e6-860b-2efdce9cbe12.png)
